src/AccessibilityInsights.Extensions.GitHub/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyTitle("AccessibilityInsights.Extensions.GitHub")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyCompany("")]
```

Comment:
We had a few AssemblyInfo.cs files where the company was set to "Microsoft". We've been told not to do that, so I changed these as I noticed them. Happy to make this a separate PR in master if there's a strong preference to handle it that way